packages/toast/src/Toast.spec.tsx

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import Toast, { ToastProps, ToastVariants } from './Toast';
+
+function renderToast(props: ToastProps) {
+  return render(<Toast {...props} />);
+}
+
+describe('packages/toast', () => {
+  describe(`when 'open' prop is`, () => {
+    test(`undefined, Toast doesn't render`, () => {
+      const { queryByRole } = renderToast({
+        body: 'hello world',
+        variant: ToastVariants.success,
+      });
+      const toast = queryByRole('status');
+      expect(toast).not.toBeInTheDocument();
+    });
+
+    test(`false, Toast doesn't render`, () => {
+      const { queryByRole } = renderToast({
+        body: 'hello world',
+        variant: ToastVariants.success,
+        open: false,
+      });
+
+      const toast = queryByRole('status');
+      expect(toast).not.toBeInTheDocument();
+    });
+
+    test('true, Toast is visible', () => {
+      const { queryByRole } = renderToast({
+        body: 'hello world',
+        variant: ToastVariants.success,
+        open: true,
+      });
+
+      const toast = queryByRole('status');
+      expect(toast).toBeInTheDocument();
+      expect(toast).toBeVisible();
```

Comment:
a bit redundant since being visible implies being in the document

Reply:
Fair point!